xcode/Subconscious/Shared/Components/Deck/DeckTheme.swift

```diff
@@ -137,7 +137,7 @@ extension String {
         // execution
         let hash = self.utf8.reduce(0) { $0 + Int($1) }
         
-        return colors[abs(hash.hashValue) % colors.count]
+        return colors[abs(hash) % colors.count]
```

Comment:
Calling `.hashValue` resulted in Swift hashing the `Int` using non-deterministic hashing 🙃 

We can just use my "cheap deterministic hash" (summing char codes) directly.